### DISCUSSION TITLE
Magic link url for staging/preview environment

### DISCUSSION THREAD

**Discussion by lucasnantonio** [supabase/supabase#3686] [Questions] [answered]
I'm using Vercel to test and deploy my app. Whenever I push to a new branch, it creates a Preview deployment with an URL like:

https://my-app-hfhqhjvnu-myname.vercel.app/

These URLs are different every time, which means my magic links break when I try to log in.

Am I missing something? Is there a way to login into preview/stating deployments like these using magic links?

Thanks in advance!

**Comment by silentworks** [comment, marked as answer]
I've answered this over here https://github.com/supabase/supabase/discussions/2760#discussioncomment-1144158, do note that you still have to do some manual work of adding the preview deployment URL to the **"Additional Redirect URLs"** each time though.
